docs/code-samples/lifecycling/daml/Scripts/FixedRateBond.daml

```diff
@@ -346,3 +352,24 @@ createParty name = do
   userId <- validateUserId name
   createUser (User userId (Some party)) [CanActAs party]
   pure party
+
+-- | Create a holding factory with reference.
+createHoldingFactory :
```

Comment:
Can we move it to a utility module? Also, same comment as for the other instance, I would not write this as a generic function